apps/base/management/commands/seed.py

```diff
@@ -0,0 +1,11 @@
+from django.core.management import BaseCommand, call_command
+
+
+class Command(BaseCommand):
+
+    help = "Seeds the database for you"
```

Comment:
instead of writing `for you`, can we please write `Seeds the database with random but sensible values`. What say @deshraj